test/verify/check-users

```diff
@@ -164,6 +164,20 @@ class TestAccounts(testlib.MachineCase):
         b.wait_visible("#accounts")
         b.wait_not_in_text('#accounts-list', "Anton Arbitrary")
 
+        m.execute("useradd --create-home barney")
+        m.execute("echo barney:foobar | chpasswd")
+        m.execute("runuser -u barney -- touch /home/barney/test.txt")
+        m.execute("test -f /home/barney/test.txt")
+
+        with b.wait_timeout(30):
+            b.wait_in_text('#accounts-list', "barney")
+        performUserAction(b, 'barney', 'Delete')
+        b.set_checked("#account-confirm-delete-files", val=True)
+        b.click('#account-confirm-delete-dialog button.apply')
+        b.wait_not_present('#account-confirm-delete-dialog')
+        m.execute("! test -f /home/barney/test.txt")
```

Comment:
Could be stronger with `test ! -e /home/barney`

Reply:
And this part was meant to test https://cockpit-logs.us-east-1.linodeobjects.com/pull-21389-0c41f814-20250109-101243-fedora-41-devel/Coverage/pkg/users/delete-account-dialog.js.gcov.html